src/app/components/constitution-page/votes/votes-grade/votes-grade.component.ts

```diff
@@ -122,34 +122,21 @@ export class VotesGradeComponent implements OnDestroy {
 	}
 
 	getSongsToVote(): Song[] {
-		let songsToVote = Array.from(this.songs.values()).filter(song => {
-			const isNotUserSong = song.user !== this.auth.uid;
-			const isAlreadyVoted = this.votes.values.has(song.id) && this.showAlreadyVoted
-			return isNotUserSong && !isAlreadyVoted && this.isSelected(song.user);
-		});
-
-		if (this.cardsSortASC) songsToVote = songsToVote.sort(compareSongASC)
-		else songsToVote = songsToVote.sort(compareSongDSC);
-
-		if (this.orderByUser) songsToVote = songsToVote.sort(compareSongUser);
-
-		if (this.orderByGrade !== GradeOrder.NONE) songsToVote = songsToVote.sort((s1, s2) => {
-			const order = this.orderByGrade === GradeOrder.INCREASE ? 1 : -1;
-
-			const g1 = this.getVote(s1) || 0;
-			const g2 = this.getVote(s2) || 0;
-
-			if (g1 > g2) return order;
-			if (g1 < g2) return -order;
-
-			return 0;
-		})
-
-		if (this.orderByFavs) songsToVote = songsToVote.sort((a, b) => {
-			if (this.isAFavorite(a)) return -1;
-			if (this.isAFavorite(b)) return 1;
-			return 0;
-		})
+		let songsToVote = Array.from(this.songs.values())
+
+		// Filter
+		songsToVote = songsToVote.filter(song => song.user !== this.auth.uid);
+		songsToVote = songsToVote.filter(song => this.votes.values.has(song.id) && this.showAlreadyVoted);
+		songsToVote = songsToVote.filter(song => this.isSelected(song.user));
+
+		// Sort
+		songsToVote.sort(compareObjectsFactory("title", this.cardsSortASC));
```

Comment:
Pourquoi sort par "title" ? Pour avoir l'ordre chronologie, il faut que ce soit sort par id.
```suggestion
		songsToVote.sort(compareObjectsFactory("id", this.cardsSortASC));
```